src/common/components/repository-input/index.js

```diff
@@ -33,6 +34,14 @@ export class RepositoryInput extends Component {
     return message;
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
I'm not sure if it's the best way to do it, but it was simplest solution I could find.

So currently it's the repository input component that redirects to `/user/repo/builds` once the repository entered is verified.

Maybe there should be a separate redux action to do that? But who should dispatch it?